integrations/great_expectations/fraud_batch_data_validation.ipynb

```diff
@@ -0,0 +1,3478 @@
+        "            \"column\":\"fraud_label\",\n",
+        "            \"value_set\": [0,1]\n",
+        "        }\n",
+        "    )\n",
+        ")\n",
+        "\n",
+        "# Assess data correctness\n",
+        "\n",
+        "expectation_suite_trans.add_expectation(\n",
+        "    ExpectationConfiguration(\n",
+        "        expectation_type=\"expect_column_min_to_be_between\",\n",
+        "        kwargs={\n",
+        "            \"column\":\"amount\",\n",
+        "            \"min_value\":0\n",
+        "        }\n",
+        "    )\n",
+        ")\n",
+        "\n",
+        "expectation_suite_trans.add_expectation(\n",
+        "    ExpectationConfiguration(\n",
+        "        expectation_type=\"expect_column_min_to_be_between\",\n",
+        "        kwargs={\n",
+        "            \"column\":\"age_at_transaction\",\n",
+        "            \"min_value\": 18,\n",
+        "            \"max_value\": 130\n",
+        "        }\n",
+        "    )\n",
+        ")\n",
+        "\n",
+        "# Or simply to monitor data statistics and quality, e.g number of null values\n",
+        "\n",
+        "expectation_suite_trans.add_expectation(\n",
+        "    ExpectationConfiguration(\n",
+        "      expectation_type=\"expect_column_mean_to_be_between\",\n",
+        "      kwargs={\"column\":\"age_at_transaction\", \"min_value\": 0, \"max_value\": 100}\n",
+        "    )\n",
+        ")\n",
+        "\n",
+        "expectation_suite_trans.add_expectation(\n",
+        "    ExpectationConfiguration(\n",
+        "        expectation_type=\"expect_column_value_to_be_null\",\n",
+        "        kwargs={\"column\":\"category\"}\n",
+        "    )\n",
+        ")\n"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "XPXAj4vTyBNn",
+        "outputId": "04dfd40a-c2f4-4beb-d053-55a222c1323e"
+      },
+      "id": "XPXAj4vTyBNn",
+      "execution_count": 47,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "{\"expectation_type\": \"expect_column_value_to_be_null\", \"kwargs\": {\"column\": \"category\"}, \"meta\": {}}"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 47
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "ge_trans_df = ge.from_pandas(trans_df, expectation_suite=expectation_suite_trans)\n",
+        "\n",
+        "validation_report_trans = ge_trans_df.validate()"
+      ],
+      "metadata": {
+        "id": "gOBpleMxLGkg"
+      },
+      "id": "gOBpleMxLGkg",
+      "execution_count": 48,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "When prototyping an expectation suite you can use Great Expectations DataFrame wrapper to evaluate expectation directly and enable auto-complete."
```

Comment:
Maybe we can add a heading here, cause it's a bit confusing, you first create an expectation suite but then in this part you don't use it and create a new one.

It's clear that it's for the other feature group, but mayube we can separate it a bit with a heading to make it more obvious.

Reply:
Ok I will make a clearer distinction